src/main/java/net/smartcosmos/dto/relationships/RelationshipLookupSpecific.java

```diff
@@ -0,0 +1,41 @@
+package net.smartcosmos.dto.relationships;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Setter;
+
+import java.beans.ConstructorProperties;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+@AllArgsConstructor
+public class RelationshipLookupSpecific {
+    @Setter(AccessLevel.NONE)
+    private String version = "1";
```

Comment:
This will be ignored by the Builder. Field initialization like this is only taken into account by the default (NoArgs) constructor which doesn't exist in this class.